spec/components/candidate_interface/course_choices_review_component_spec.rb

```diff
@@ -58,6 +58,29 @@
         expect(result.css('.app-summary-card__title').text).to include(course_choice.provider.name)
       end
     end
+
+    context 'when course choice is single site' do
+      it 'renders without the "Change" location links' do
+        result = render_inline(described_class.new(application_form: application_form))
+
+        expect(result.css('.govuk-summary-list__actions').text).not_to include('Change')
+      end
+    end
```

Comment:
Think you might need `FeatureFlag.activate('edit_course_choices')` for this, right?